src/plugins/core/image.ts

```diff
@@ -104,6 +112,25 @@ export class ImagePlugin extends CorePlugin<ImageState> implements ImageState {
     };
     this.dispatch("CREATE_FIGURE", { sheetId, figure });
   }
+
+  import(data: WorkbookData) {
+    for (const sheet of data.sheets) {
+      const images = (sheet.figures || []).filter((figure) => figure.tag === "image");
+      for (const image of images) {
+        this.history.update("nextId", this.nextId + 1);
```

Comment:
There's the same kind of trick for charts.
It seems vaguely louche, fragile and working by chance (if it works), right?

Reply:
I really don't know how we could avoid this actually. having a UI duplicate sheet command that would allow the dispatching of other commands ? one of them could be the current duplicate sheet (which would handle cells and stuff) and another one could be DUPLICATE_CHART/ DUPLICATE_IMAGE ?

Reply:
It's louche and fragile, I create a [task](https://www.odoo.com/web#id=3099841&cids=1&menu_id=4720&action=333&active_id=2328&model=project.task&view_type=form) to handle this.